src/arrayMethodSort.js

```diff
@@ -3,10 +3,38 @@
 /**
  * Implement method Sort
  */
-function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
-  };
-}
+const applyCustomSort = (a, b) => {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  } else if (stringA === stringB) {
+    return 0;
+  } else {
+    return -1;
+  }
+};
+
+[].__proto__.sort2 = function(callback = applyCustomSort) {
+  let count;
```

Comment:
Why won't you write count = 0 at this moment? Then there's no need to write it in do-while